recipe/recipe_test.go

```diff
@@ -68,11 +69,10 @@ func TestGetRecipe(t *testing.T) {
 
 		Convey("Then whe GetRecipe is called, one GET /recipes/ID call is performed and the expected error is returned", func() {
 			recipe, err := recipeClient.GetRecipe(ctx, testUserAuthToken, testServiceToken, recipeID)
-			So(err, ShouldResemble, &Error{
-				err:        errors.New("failed to unmarshal error response body: invalid character 'r' looking for beginning of value"),
-				statusCode: http.StatusBadRequest,
-				logData:    log.Data{"response_body": "[response body empty]"},
-			})
+			So(err, ShouldResemble, dperrors.New(
+				errors.New(""),
+				http.StatusBadRequest,
+				nil))
```

Comment:
last bracket should be closed on the next line (keep in line with where scope was opened)